src/main/java/com/paneedah/mwc/proxies/CommonProxy.java

```diff
@@ -131,4 +130,4 @@ public void init(final ModernWarfareMod mod, final CompatibleFmlInitializationEv
 
     public void postInit(final ModernWarfareMod mod, final FMLPostInitializationEvent initializationEvent) {
     }
-}
\ No newline at end of file
+}
```

Comment:
Also could you please turn on the IntelliJ setting to automatically set a new line at the end of each file